lib/enum/arrays.rb

```diff
@@ -1,24 +1,31 @@
 class Array
 
   def sum_of_numeric_elements
+    select{|num| num.is_a? Numeric}.inject(:+)
   end
 
   def product_of_numeric_elements
+    select{|element| element.is_a? Numeric}.inject(:*)
   end
 
   def even_numeric_elements
+    select{|number| (number.is_a? Fixnum) && (number % 2 == 0)}
```

Comment:
This works, and Ruby also has a built-in method called .even? (and .odd? for the next problem)